aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/AmazonPinpointAnalyticsPlugin.java

```diff
@@ -120,16 +120,29 @@ public void recordEvent(@NonNull AnalyticsEvent analyticsEvent)
      * {@inheritDoc}
      */
     @Override
-    public void registerGlobalProperties(@NonNull Properties properties) {
-        throw new UnsupportedOperationException("This operation has not been implemented yet.");
+    public void registerGlobalProperties(@NonNull Properties properties) throws AnalyticsException {
```

Comment:
registering global properties is synchronous call that does not make a network call or any background processing, so we don't need a callback, but we can consider logging and notifying through hub rather than throwing an exception in case of failure. I'll check ios behavior with Michael.